src/main/java/com/sleekbyte/tailor/format/JSONFormatter.java

```diff
@@ -0,0 +1,87 @@
+package com.sleekbyte.tailor.format;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.sleekbyte.tailor.common.ColorSettings;
+import com.sleekbyte.tailor.common.ExitCode;
+import com.sleekbyte.tailor.common.Messages;
+import com.sleekbyte.tailor.output.ViolationMessage;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Formatter that displays violation messages in valid JSON output.
+ */
+public final class JSONFormatter extends Formatter {
+
+    private Gson gson = new GsonBuilder().disableHtmlEscaping().setPrettyPrinting().create();
```

Comment:
Is there a reason we are instantiating this here or can we instantiate this in the constructor for JSONFormatter?